tests/test_octodns_source_envvar.py

```diff
@@ -1,6 +1,12 @@
 from six import text_type
 from unittest import TestCase
-from unittest.mock import patch
+
+try:
+    # Python 3
+    from unittest.mock import patch
+except ImportError:
+    # Python 2
+    from mock import patch
```

Comment:
I'm not seeing any problems importing `from mock import patch` in python 2 or 3, we're installing the standalone `mock` library which should make it available as `mock` for both.

Reply:
This was the test:

```
ERROR: Failure: ImportError (No module named mock)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/blanariu/work/cst/octodns/env/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/Users/blanariu/work/cst/octodns/env/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/Users/blanariu/work/cst/octodns/env/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/Users/blanariu/work/cst/octodns/tests/test_octodns_source_envvar.py", line 3, in <module>
    from unittest.mock import patch
ImportError: No module named mock
```

Reply:
For this one was `requirements-dev.txt` installed? Fairly sure that should cover getting `mock` installed in all cases.

```console
coho:octodns ross$ grep mock requirements-dev.txt
mock
requests_mock
```